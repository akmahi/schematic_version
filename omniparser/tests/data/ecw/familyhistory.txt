apuid|RecordID|PatientId|PatientAccountNo|EncounterId|ProviderNPI|Relationship|Status|AGE|DateOfBirth|IcdItemid|ICD_Code|ICD_Desc|DiagnosedYear|DiagnosedAge
305922||526642|526642|2419095|1679623896|Paternal Grand Father|alive|||||||
305922||526642|526642|2419095|1679623896|Father|alive|||||||
305922||526642|526642|2419095|1679623896|Mother|alive|||||||
305922||9977|400317|2517824|1679623896|Father|unknown|||||||
305922||9977|400317|2517824|1679623896|Mother|deceased|||||||
305922||13532|520235|2537450|1679623896|Father|alive|||||||
305922||13532|520235|2537450|1679623896|Mother|alive|||||||
305922||531301|531301|2540717|1679623896|Father|unknown|||||||
305922||531301|531301|2540717|1679623896|Mother|alive|||||||
305922|305922.11582.2546108.10688.110448|11582|508104|2546108|1679623896|Father|deceased|||110448|429.9|Cardiopathy||
305922||11582|508104|2546108|1679623896|Mother|alive|||||||
305922||11069|502825|2551331|1679623896|Father|unknown|||||||
305922||11069|502825|2551331|1679623896|Mother|alive|||||||
305922||529253|529253|2510419|1679623896|Father|alive|||||||
305922||529253|529253|2510419|1679623896|Mother|alive|||||||
305922||540261|540261|2527186|1679623896|Father|alive|||||||
305922||540261|540261|2527186|1679623896|Mother|alive|||||||
305922||532178|532178|2531535|1679623896|Father|alive|||||||
305922||532178|532178|2531535|1679623896|Mother|alive|||||||
305922||10085|400681|2544650|1679623896|Father|deceased|||||||
305922||10085|400681|2544650|1679623896|Mother|deceased|||||||
305922||9321|100047|2517560|1679623896|Father|deceased|||||||
305922||9321|100047|2517560|1679623896|Mother|alive|||||||
305922||9720|200019|2542571|1679623896|Siblings|alive|||||||
305922||9720|200019|2542571|1679623896|Father|deceased|||||||
305922||9720|200019|2542571|1679623896|Mother|alive|||||||
305922||536047|536047|2547790|1679623896|Children|alive|||||||
305922||536047|536047|2547790|1679623896|Father|alive|||||||
305922||536047|536047|2547790|1679623896|Mother|deceased|||||||
