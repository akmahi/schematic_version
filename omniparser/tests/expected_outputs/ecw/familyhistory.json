[
	{
		"family_history": {
			"encounter": {
				"id": "2419095"
			},
			"patient": {
				"account_no": "526642",
				"id": "526642"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Paternal Grand Father",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2419095"
			},
			"patient": {
				"account_no": "526642",
				"id": "526642"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2419095"
			},
			"patient": {
				"account_no": "526642",
				"id": "526642"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2517824"
			},
			"patient": {
				"account_no": "400317",
				"id": "9977"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "unknown"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2517824"
			},
			"patient": {
				"account_no": "400317",
				"id": "9977"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "deceased"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2537450"
			},
			"patient": {
				"account_no": "520235",
				"id": "13532"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2537450"
			},
			"patient": {
				"account_no": "520235",
				"id": "13532"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2540717"
			},
			"patient": {
				"account_no": "531301",
				"id": "531301"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "unknown"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2540717"
			},
			"patient": {
				"account_no": "531301",
				"id": "531301"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2546108"
			},
			"icd_code": "429.9",
			"icd_desc": "Cardiopathy",
			"icd_item_id": "110448",
			"patient": {
				"account_no": "508104",
				"id": "11582"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"record_id": "305922.11582.2546108.10688.110448",
			"relationship": "Father",
			"status": "deceased"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2546108"
			},
			"patient": {
				"account_no": "508104",
				"id": "11582"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2551331"
			},
			"patient": {
				"account_no": "502825",
				"id": "11069"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "unknown"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2551331"
			},
			"patient": {
				"account_no": "502825",
				"id": "11069"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2510419"
			},
			"patient": {
				"account_no": "529253",
				"id": "529253"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2510419"
			},
			"patient": {
				"account_no": "529253",
				"id": "529253"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2527186"
			},
			"patient": {
				"account_no": "540261",
				"id": "540261"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2527186"
			},
			"patient": {
				"account_no": "540261",
				"id": "540261"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2531535"
			},
			"patient": {
				"account_no": "532178",
				"id": "532178"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2531535"
			},
			"patient": {
				"account_no": "532178",
				"id": "532178"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2544650"
			},
			"patient": {
				"account_no": "400681",
				"id": "10085"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "deceased"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2544650"
			},
			"patient": {
				"account_no": "400681",
				"id": "10085"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "deceased"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2517560"
			},
			"patient": {
				"account_no": "100047",
				"id": "9321"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "deceased"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2517560"
			},
			"patient": {
				"account_no": "100047",
				"id": "9321"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2542571"
			},
			"patient": {
				"account_no": "200019",
				"id": "9720"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Siblings",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2542571"
			},
			"patient": {
				"account_no": "200019",
				"id": "9720"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "deceased"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2542571"
			},
			"patient": {
				"account_no": "200019",
				"id": "9720"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2547790"
			},
			"patient": {
				"account_no": "536047",
				"id": "536047"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Children",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2547790"
			},
			"patient": {
				"account_no": "536047",
				"id": "536047"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Father",
			"status": "alive"
		}
	},
	{
		"family_history": {
			"encounter": {
				"id": "2547790"
			},
			"patient": {
				"account_no": "536047",
				"id": "536047"
			},
			"practice": {
				"apuid": "305922"
			},
			"provider": {
				"npi": "1679623896"
			},
			"relationship": "Mother",
			"status": "deceased"
		}
	}
]
